### PR TITLE
Updating SLA from 1-2 bus days to 3-5 bus days.

### PR DESCRIPTION
**Pull request by ssmith-dbt**
## What are you changing in this pull request and why?

Updating the SLA on private link setup to more accurately reflect turn around.

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.

### CODE REVIEW

**Comment by natty-li** [comment]
I noticed there wasn't a similar note on the `databricks` set up page - https://docs.getdbt.com/docs/cloud/secure/databricks-privatelink. Should we add something for that too for consistency's sake?

**Review by natty-li — APPROVED** [review]

File: website/docs/docs/cloud/secure/redshift-privatelink.md [hunk 1]
```diff
@@ -102,7 +105,7 @@ Subject: New Multi-Tenant PrivateLink Request
 - dbt Cloud multi-tenant environment (US, EMEA, AU):
 ```
 
-dbt Labs will work on your behalf to complete the PrivateLink setup. Please allow 1-2 business days for this process to complete. Support will contact you when the endpoint is available.
+dbt Labs will work on your behalf to complete the PrivateLink setup. Please allow 3-5 business days for this process to complete. Support will contact you when the endpoint is available.
```
Comment (runleonarun) [inline website/docs/docs/cloud/secure/redshift-privatelink.md]:
```suggestion
import PrivateLink from '/snippets/_PrivateLink-SLA.md';

<PrivateLink />
```

File: website/docs/docs/cloud/secure/databricks-privatelink.md [hunk 2]
```diff
@@ -23,6 +23,12 @@ Subject: New Multi-Tenant PrivateLink Request
 - Databricks cluster AWS Region (e.g., us-east-1, eu-west-2):
 - dbt Cloud multi-tenant environment (US, EMEA, AU):
 ```
+
+import PrivateLink from '/snippets/_PrivateLink-SLA.md';
+
+<PrivateLink />
+
```
Comment (runleonarun) [inline website/docs/docs/cloud/secure/databricks-privatelink.md:28]:
```suggestion
```

**Comment by runleonarun** [comment]
Thanks team! I moved the content into a reusable snippet to reduce the maintenance burden and keep wording consistent throughout. I will verify in the latest build then merge if it's ready.

File: website/docs/docs/cloud/secure/databricks-privatelink.md [hunk 3]
```diff
@@ -23,6 +23,11 @@ Subject: New Multi-Tenant PrivateLink Request
 - Databricks cluster AWS Region (e.g., us-east-1, eu-west-2):
 - dbt Cloud multi-tenant environment (US, EMEA, AU):
 ```
+
+import PrivateLink from '/snippets/_PrivateLink-SLA.md';
```
Comment (runleonarun) [inline website/docs/docs/cloud/secure/databricks-privatelink.md]:
```suggestion
import PrivateLinkSLA from '/snippets/_PrivateLink-SLA.md';
```

File: website/docs/docs/cloud/secure/databricks-privatelink.md [hunk 4]
```diff
@@ -23,6 +23,11 @@ Subject: New Multi-Tenant PrivateLink Request
...
+
+<PrivateLink />
```
Comment (runleonarun) [inline website/docs/docs/cloud/secure/databricks-privatelink.md]:
```suggestion
<PrivateLinkSLA />
```

File: website/docs/docs/cloud/secure/redshift-privatelink.md [hunk 5]
```diff
@@ -102,7 +104,9 @@ Subject: New Multi-Tenant PrivateLink Request
 - dbt Cloud multi-tenant environment (US, EMEA, AU):
 ```
 
-dbt Labs will work on your behalf to complete the PrivateLink setup. Please allow 1-2 business days for this process to complete. Support will contact you when the endpoint is available.
+import PrivateLinkSLA from '/snippets/_PrivateLink-SLA.md';
```
Comment (runleonarun) [inline website/docs/docs/cloud/secure/redshift-privatelink.md]: Already imported
```suggestion
```

File: website/docs/docs/cloud/secure/redshift-privatelink.md [hunk 6]
```diff
@@ -102,7 +104,8 @@ Subject: New Multi-Tenant PrivateLink Request
 - dbt Cloud multi-tenant environment (US, EMEA, AU):
 ```
 
-dbt Labs will work on your behalf to complete the PrivateLink setup. Please allow 1-2 business days for this process to complete. Support will contact you when the endpoint is available.
+
```
Comment (runleonarun) [inline website/docs/docs/cloud/secure/redshift-privatelink.md]:
```suggestion
```

**Review by runleonarun — APPROVED** [review]

**Pull request merged**